lib/ransack/adapters/active_record/ransack/context.rb

```diff
@@ -37,15 +37,14 @@ def initialize(object, options = {})
         @base = @join_dependency.join_base
         @engine = @base.arel_engine
       end
+    end
 
-      @default_table = Arel::Table.new(
-        @base.table_name, as: @base.aliased_table_name, type_caster: self
-        )
```

Comment:
This variable didn't seem to be used anywhere and it represented a lot of memory usage, so I removed it.